src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Basic.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Basic
+    {
+        private static readonly byte[] ExtraArray = Encoding.UTF8.GetBytes("ExtraArray");
+        private static readonly byte[] First = Encoding.UTF8.GetBytes("first");
+        private static readonly byte[] Last = Encoding.UTF8.GetBytes("last");
+        private static readonly byte[] Age = Encoding.UTF8.GetBytes("age");
+        private static readonly byte[] PhoneNumbers = Encoding.UTF8.GetBytes("phoneNumbers");
+        private static readonly byte[] Address = Encoding.UTF8.GetBytes("address");
+        private static readonly byte[] Street = Encoding.UTF8.GetBytes("street");
+        private static readonly byte[] City = Encoding.UTF8.GetBytes("city");
+        private static readonly byte[] Zip = Encoding.UTF8.GetBytes("zip");
+
+        private const int DataSize = 10;
+
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+        private PooledBufferWriter<byte> _pooledBufferWriter;
+
+        private int[] _numberArrayValues;
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        [Params(true, false)]
+        public bool Pool;
```

Comment:
Is it OK to have 2*2*2*2 = 16 perf test, like this.

If this is a concern, I could potentially remove the `Pool` params and reduce the test count down to 8, though I would prefer to keep it.

The total benchmarks (136 total tests) take around 10-15 minutes to complete on my machine.

Reply:
I think the intend here is to (micro) benchmark the new JSON API, not the array pool (which should be a different set of benchmarks).
If we want to test an full RWC E2E JSON scenario, then they should be a different set of benchmarks.

Reply:
agreed. Unless somehow the Json api behaves differently depending on where it is writing to, I would remove any use of the array pool. Keeping benchmarks as tight and targeted as possible is great.

Reply:
> If we want to test an full RWC E2E JSON scenario, then they should be a different set of benchmarks.

That's a fair point. Will remove it then.